sdk/core/src/request.rs

```diff
@@ -0,0 +1,60 @@
+use http::{HeaderMap, Method};
+use std::fmt::Debug;
+use url::Url;
+
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct Request {
+    pub url: Url,
+    pub method: Method,
+    pub headers: HeaderMap,
+    pub payload: bytes::Bytes,
+}
+
+impl Request {
+    #[allow(dead_code)]
```

Comment:
Why?

Reply:
Since we don't have - yet - any other custom policy mutability is not needed. I've left `mut` there to signal that the vector should be `mut` but it's not (temporarily). I can remove it!